tests/test_http_client.py

```diff
@@ -603,7 +612,7 @@ def test_no_retry_request_stream_error(
         self, mock_retry, response, check_call_numbers
     ):
         mock_retry(no_retry_error_num=self.max_retries())
-        with pytest.raises(stripe.error.APIConnectionError):
+        with pytest.raises(stripe.APIConnectionError):
```

Comment:
nit: consider importing the APIConnectionError type directly

Reply:
Done!